### PR TITLE
feat: added scyllaDB for go gin server

### PR DESCRIPTION
**Pull request by azar-writes-code**
Hi @mahendraintelops ,

This PR contains the template for scylla DB integration for go-gin-server.

### CODE REVIEW

File: frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl [hunk 1]
```diff
@@ -145,7 +145,7 @@ func ({{.SmallResourceNameSingular}}Controller *{{.CapsResourceNameSingular}}Con
 		return
 	}
 
-	context.JSON(http.StatusNoContent, gin.H{})
+	context.JSON(http.StatusOK, gin.H{"message": "{{.CapsResourceNameSingular}} deleted successfully"})
```
Comment (mahendraintelops) [inline frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl]: Please keep this what it was before

File: frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl [hunk 2]
```diff
@@ -125,7 +125,7 @@ func ({{.SmallResourceNameSingular}}Controller *{{.CapsResourceNameSingular}}Con
 		return
 	}
 
-	context.JSON(http.StatusNoContent, gin.H{})
+	context.JSON(http.StatusOK, gin.H{"message": "{{.CapsResourceNameSingular}} updated successfully"})
```
Comment (mahendraintelops) [inline frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl]: Please keep this what it was before

**Review by mahendraintelops — DISMISSED** [review]

File: frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl (hunk 1 above)
Comment (azar-writes-code) [reply frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl]: Sure @mahendraintelops

File: frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl (hunk 2 above)
Comment (azar-writes-code) [reply frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl]: Sure @mahendraintelops

Comment (azar-writes-code) [reply frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl]: Done with changes @mahendraintelops

File: frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl (hunk 1 above)
Comment (azar-writes-code) [reply frameworks/go-gin-server/pkg/rest/server/controllers/sqls-controller.go.tmpl]: Done @mahendraintelops

File: frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-controller.go.tmpl [hunk 3]
```diff
@@ -117,7 +117,8 @@ func ({{.SmallResourceNameSingular}}Controller *{{.CapsResourceNameSingular}}Con
 		return
 	}
 
-	context.JSON(http.StatusNoContent)
+	context.JSON(http.StatusOK, gin.H{"message": "{{.CapsResourceNameSingular}} updated successfully"})
```
Comment (mahendraintelops) [inline frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-controller.go.tmpl:120]: please dont change this and keep the same as it was earlier

File: frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-controller.go.tmpl [hunk 4]
```diff
@@ -138,7 +139,7 @@ func ({{.SmallResourceNameSingular}}Controller *{{.CapsResourceNameSingular}}Con
 		return
 	}
 
-	context.JSON(http.StatusNoContent)
+	context.JSON(http.StatusOK, gin.H{"message": "{{.CapsResourceNameSingular}} deleted successfully"})
```
Comment (mahendraintelops) [inline frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-controller.go.tmpl:142]: please dont change this and keep the same as it was earlier

File: frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-column-based-controller.go.tmpl [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  id := context.Param("id")
+
+	// trigger {{.SmallResourceNameSingular}} deletion
+	if err := {{.SmallResourceNameSingular}}Controller.{{.SmallResourceNameSingular}}Service.Delete{{.CapsResourceNameSingular}}(id); err != nil {
+		log.Error(err)
+		context.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+
+	context.JSON(http.StatusOK, gin.H{"message": "{{.CapsResourceNameSingular}} deleted successfully"})
```
Comment (mahendraintelops) [inline frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-column-based-controller.go.tmpl]: please add the same response as its for other delete methods

File: frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-column-based-controller.go.tmpl [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+	}
+
+	// trigger {{.SmallResourceNameSingular}} update
+	if _, err := {{.SmallResourceNameSingular}}Controller.{{.SmallResourceNameSingular}}Service.Update{{.CapsResourceNameSingular}}(id,&input); err != nil {
+		log.Error(err)
+		context.JSON(http.StatusInternalServerError, gin.H{"error": err.Error()})
+		return
+	}
+
+	context.JSON(http.StatusOK, gin.H{"message": "{{.CapsResourceNameSingular}} updated successfully"})
```
Comment (mahendraintelops) [inline frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-column-based-controller.go.tmpl]: Please add the response here too

**Review by mahendraintelops — CHANGES_REQUESTED** [review]

File: frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-controller.go.tmpl (hunk 3 above)
Comment (azar-writes-code) [reply frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-controller.go.tmpl:120]: Hi @mahendraintelops, Changes are made as suggested, Please let me know if anything needed

File: frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-controller.go.tmpl (hunk 4 above)
Comment (azar-writes-code) [reply frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-controller.go.tmpl:142]: Hi @mahendraintelops, Changes are made as suggested, Please let me know if anything needed

File: frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-column-based-controller.go.tmpl (hunk 5 above)
Comment (azar-writes-code) [reply frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-column-based-controller.go.tmpl]: Hi @mahendraintelops, Changes are made as suggested, Please let me know if anything needed

File: frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-column-based-controller.go.tmpl (hunk 6 above)
Comment (azar-writes-code) [reply frameworks/go-gin-server/pkg/rest/server/controllers/nosqls-column-based-controller.go.tmpl]: Hi @mahendraintelops, Changes are made as suggested, Please let me know if anything needed

**Pull request closed without merging**